### DISCUSSION TITLE
Should be possible to request reversion of a specific change or set of changes.

### DISCUSSION THREAD

**Discussion by vectro** [theory/sqitch#661] [Ideas]
It seems to me, as someone who is new to Sqitch, that the most common use case for reverting a change is, "I just deployed a change and it's not working the way that I want and now I want to revert it".

In that situation, it is somewhat painful that there is no way to ask Sqitch to revert the thing that was just deployed. Instead one must find a prior change and revert to that.

It might be helpful if one can specify a list of changes to be reverted. If the list is not possible (for example, because subsequent dependent changes have been deployed), it would always be OK to fail with a reasonable error message instead.

**Comment by theory** [comment]
Usually, in my experience, this is the last change deployed. For that, you can specify the change as `@HEAD^`:

``` sh
sqitch revert @HEAD^
```

In fact, a key part of my development processes, as I'm working on a change is [rebasing](http://sqitch.org/docs/manual/sqitch-rebase/) it:

``` sh
sqitch rebase @HEAD^
```

Have a look at the [sqitch-changes](http://sqitch.org/docs/manual/sqitchchanges/) guide for more on how to specify changes other than by name.
